.github/COMMIT_CONVENTION.md

```diff
@@ -0,0 +1,51 @@
+## Git Commit Message Convention
+
+> This is adapted from [Angular's commit convention](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular).
+
+#### TL;DR:
+
+Messages must be matched by the following regex:
+
+<!-- prettier-ignore -->
+```js
+/^(revert: )?(feat|fix|docs|refactor|perf|test|ci|chore|release)(\(.+\))?: .{1,50}/
+```
+
+#### Examples
+
+Appears under "Features" header:
```

Comment:
`Let's say you added a new feature to the Button component, which is it's own package "@material-svelte/button". You would write your commit message like this (of course changing the last part):`